rotkehlchen/fval.py

```diff
@@ -51,6 +51,9 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return f'FVal({str(self.num)})'
 
+    def __hash__(self) -> int:
```

Comment:
this is needed because of using an FVal as a key somewhere?

Reply:
Yes, I used FVal as a key in a dict for detection of relevant trades
![image](https://user-images.githubusercontent.com/63492346/221597649-b1a862fa-87de-44db-b703-cc9a39d37e46.png)